### PR TITLE
v1.1 backport custom error rename

### PR DESCRIPTION
**Pull request by CriesofCarrots**
Also remove librapay merge-conflict artifacts, which offend my sensibilities

### CODE REVIEW

File: programs/librapay/src/librapay_transaction.rs [hunk 1]
```diff
@@ -161,16 +161,8 @@ mod tests {
     fn create_bank(lamports: u64) -> (Arc<Bank>, Keypair, Keypair, Pubkey, Pubkey) {
         let (mut genesis_config, mint) = create_genesis_config(lamports);
         genesis_config.rent.lamports_per_byte_year = 0;
-<<<<<<< HEAD
```
Comment (jackcmay) [inline programs/librapay/src/librapay_transaction.rs:164]: @CriesofCarrots Merge conflicts?

Comment (CriesofCarrots) [reply programs/librapay/src/librapay_transaction.rs:164]: Right, I'm removing those. Those went into the code base here (https://github.com/solana-labs/solana/pull/9818) and here (https://github.com/solana-labs/solana/pull/9814)

Comment (jackcmay) [reply programs/librapay/src/librapay_transaction.rs:164]: Heh, thanks, I need to re-enable libra sooner than later to avoid this kind of bit rot

Comment (jackcmay) [reply programs/librapay/src/librapay_transaction.rs:164]: I can remove them on master

Comment (CriesofCarrots) [reply programs/librapay/src/librapay_transaction.rs:164]: I don't think they are on master

Comment (jackcmay) [reply programs/librapay/src/librapay_transaction.rs:164]: Nevermind, just got put in mergify on v1.1

**Comment by CriesofCarrots** [comment]
@jackcmay , does this look good now?

**Comment by mvines** [comment]
lgtm

**Pull request merged**